xbdd-web/src/modules/report/FeatureListContainer/TagViewFeatureList/styles/TagListStyles.js

```diff
@@ -0,0 +1,44 @@
+const tagListStyles = {
+  xbddTagList: {
+    borderRadius: "0 0 4px 4px",
+    height: "100%",
+    overflow: "scroll",
+    position: "relative",
+    width: "100%",
+  },
+  xbddFilterButton: {
+    width: 90,
```

Comment:
While this is valid can we change it to "90px" to be consistent

Reply:
Just found that it is not used at all, so I deleted it.